### PR TITLE
Hosting on Clodui

### PR DESCRIPTION
**Pull request by cvrajeesh**
Documentation which explains how to host Hugo websites on [Clodui](https://www.clodui.com)

### CODE REVIEW

File: content/en/hosting-and-deployment/hosting-on-clodui.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+   ```
+5. New changes to the website can be deployed by running following command from the terminal
+
+   ```
+   clodui website deploy \
+        --source-dir ./public \
+        --website-id <website id>
+   ```
+
+#### Continuos deployment from GitHub actions
```
Comment (coliff) [inline content/en/hosting-and-deployment/hosting-on-clodui.md:70]: continuous Typo

Comment (cvrajeesh) [reply content/en/hosting-and-deployment/hosting-on-clodui.md]: thank you @coliff, fixed typos and updated my PR.

What do you think about integrating  https://github.com/marketplace/typo-ci to automatically detect typos?

Comment (coliff) [reply content/en/hosting-and-deployment/hosting-on-clodui.md]: Yes - I think it'd be a great idea to add something like that... or add cSpell checking or similar
https://github.com/streetsidesoftware/cspell#readme

**Comment by coliff** [comment]
clodui isn't a common/popular platform right now (9 followers on Twitter) so I personally don't think we should add this page to the docs for now,

**Comment by cvrajeesh** [comment]
I didn't know that popularity is a criterion to my mind it's all about providing choice to users. You may close the PR if it doesn't meet your standards.

**Comment by coliff** [comment]
I'm just helping out around here with docs, lets see what @bep says

**Pull request closed without merging**